include/igl/opengl/MeshGL.h

```diff
@@ -81,6 +85,25 @@ class MeshGL
   RowMatrixXf points_V_vbo;
   RowMatrixXf points_V_colors_vbo;
 
+  // Text Rendering
+  typedef struct 
```

Comment:
Don't need typedef (this isn't C). `struct TextGL { ... };` will work just as well.